superchain/configs/goerli-dev-0/superchain.yaml

```diff
@@ -1,7 +1,7 @@
 name: Goerli Dev 0
 l1:
   chain_id: 5
-  public_rpc: https://ethereum-goerli-rpc.allthatnode.com
+  public_rpc: wss://goerli.gateway.tenderly.co
```

Comment:
Might be something to pinch off this PR on to a separate one, if we think it makes sense. I think the existing RPC endpoint is deprecated.